pom.xml

```diff
@@ -135,6 +154,12 @@
         <artifactId>commons-validator</artifactId>
         <version>${commons-validator.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.junit.platform</groupId>
```

Comment:
Move this down with the rest of the JUnit dependencies and remove the test `scope`